### PR TITLE
Support for kernel-v2 and add new metric

### PR DESCRIPTION
**Pull request by mogproject**

### CODE REVIEW

File: src/algs/kernel.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def remove_arbitrary(A, blocks, k, kernel_v2_enabled):
     # reduction rule 2
     toremove = []
     b_index=0
+    num_reduced_blocks = 0  # the number of blocks the reduction rule was applicable to
```
Comment (mogproject) [inline src/algs/kernel.py:45]: new metric

File: src/exp_runkernel.py [hunk 2]
```diff
@@ -258,6 +259,8 @@ def main():
         help="enter first seed value in seed range", required=True)
     parser.add_argument('-l', '--last_seed', type=int,
         help="enter last seed value in seed range", required=True)
+    parser.add_argument('--v2', action='store_true',
```
Comment (mogproject) [inline src/exp_runkernel.py:262]: new argument for the testing script

**Comment by mogproject** [comment]
See the following for the expected output.
https://github.com/TheoryInPractice/ewcc-kernel/wiki/Metrics

**Comment by sjain12** [comment]
Looks good

**Pull request merged**